events_test/src/main/java/com/adobe/aio/event/management/RegistrationServiceTester.java

```diff
@@ -61,19 +64,40 @@ public Registration createJournalRegistration(String registrationName,
           .addEventsOfInterests(getTestEventsOfInterestBuilder(providerId, eventCode).build()));
   }
 
-    public Registration createRegistration(
+  public Registration createRuntimeWebhookRegistration(String registrationName, String providerId,
+      String eventCode, String runtimeAction) {
+    return createRegistration(RegistrationCreateModel.builder()
+        .name(registrationName)
+        .description(TEST_DESCRIPTION)
+        .deliveryType(DELIVERY_TYPE_WEBHOOK)
+        .runtimeAction(runtimeAction)
+        .addEventsOfInterests(getTestEventsOfInterestBuilder(providerId, eventCode).build()));
+  }
+
+  public Registration createRegistration(
       RegistrationCreateModel.Builder registrationInputModelBuilder) {
     RegistrationCreateModel registrationInputModel =
         registrationInputModelBuilder.clientId(this.workspace.getApiKey()).build();
     Optional<Registration> registration = registrationService.createRegistration(registrationInputModelBuilder);
     assertTrue(registration.isPresent());
-    Registration registratinCreated = registration.get();
+    Registration registrationCreated = registration.get();
     logger.info("Created AIO Event Registration: {}", registration.get());
-    String registrationId = registratinCreated.getRegistrationId();
+    String registrationId = registrationCreated.getRegistrationId();
     assertNotNull(registrationId);
-    assertEquals(registrationInputModel.getDescription(), registratinCreated.getDescription());
-    assertEquals(registrationInputModel.getName(), registratinCreated.getName());
-    assertEquals(registrationInputModel.getDeliveryType(), registratinCreated.getDeliveryType());
+    assertEquals(registrationInputModel.getDescription(), registrationCreated.getDescription());
+    assertEquals(registrationInputModel.getName(), registrationCreated.getName());
+    assertEquals(registrationInputModel.getDeliveryType(), registrationCreated.getDeliveryType());
+
+    if (!registrationInputModel.getDeliveryType().equals(DELIVERY_TYPE_JOURNAL)) {
```

Comment:
Shouldn't we check for:
```suggestion
    if (registrationInputModel.getDeliveryType().equals(DELIVERY_TYPE_WEBHOOK)) {
```

Reply:
good catch ! it the registration uses runtime, it cannot be a journal delivery